autoscheduler/user_sessions/tests/api_tests.py

```diff
@@ -0,0 +1,87 @@
+from rest_framework.test import APITestCase
+from django.contrib.sessions.models import Session
+
+class APITests(APITestCase):
+    """ Tests functionality of the sessions api """
+    def tearDown(self):
+        """ Delete sessions table after each test so logging in creates a new session """
+        Session.objects.all().delete()
+
+    def test_set_last_term_gives_valid_response_non_empty(self):
+        """ Tests that /sessions/set_last_term updates correctly
+            with a valid term code
+        """
+        # Arrange
+        expected_term = '202031'
+        # login() is necessary for test client to use sessions
+        self.client.login()
```

Comment:
Yeah, `login()` is usually used with the credentials to provide authorization, but it's being used here so that Django creates a session for requests that the client makes. If you remove it, `request.session` won't be set and so the tests won't work